spec/features/user/user_can_browse_spec.rb

```diff
@@ -3,10 +3,10 @@
 describe 'As a User' do
   before :each do
     @user = create(:user)
-    @podcast = create(:podcast, user: @user, location: "Denver", name: 'podcast1', description: 'description1')
-    @podcast2 = create(:podcast, user: @user, location: "NYC", name: 'podcast2', description: 'description2')
-    @podcast3 = create(:podcast, user: @user, location: "Denver", name: 'podcast3', description: 'description3')
-    @podcast4 = create(:podcast, user: @user, location: "Dallas", name: 'podcast11', description: 'description4', adult_content: true)
+    @podcast = create(:podcast, user: @user, location: "Denver", name: 'podcast1', description: 'description1', active: true)
```

Comment:
Creating the scope for Podcast to be active: true broke our original tests so we added active:true to the spec.